### ISSUE TITLE
NVDA cannot read Greek 

### ISSUE THREAD

**Issue by Chana-Miryam** [nvaccess/nvda#13310]
It spells the words with an English accent. It cannot read any Greek text.

**Comment by CyrilleB79** [comment]
Hello @Chana-Miryam

Your issue description lacks information. For the next time please, follow the issue template when opening a [new issue](https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&template=bug_report.md).

Which voice synthesizer are you using?
For NVDA to read Greek, you need to use a Greek synthesizer. ESpeak, which is bundled in NVDA provides two voices for Greek: "Greek" (modern) and "Greek ancient". For Windows 10 and above, there are also OneCore voices and [this page](https://support.microsoft.com/en-us/windows/appendix-a-supported-languages-and-voices-4486e345-7730-53da-fcfe-55cc64300f01) indicates that one is available for Greek.

HTH

**Comment by Chana-Miryam** [comment]
I have just started using it and I am struggling... I can't answer all these questions in the template...

**Comment by CyrilleB79** [comment]
OK so let's begin progressively with easier questions:
Why are you using NVDA? Are you a blind user? Are you a website developer or tester interested in accessibility?
E

**Comment by CyrilleB79** [comment]
@Chana-Miryam, the information of the issue template are still lacking and you have not answered to my last attempt with easier questions.

Are you still facing this issue and are you still interested in it?

Without any feedback from your side, NVAccess will probably have to close this issue and reopen it when more information is provided.
Cc @seanbudd, @feerrenrut.

Note also the following related issues:
* #13882: seems a more recent duplicate of this issue, but with more information and the template completed.
* #1476 (closed), on the same topic.

**Comment by seanbudd** [comment]
Closing as duplicate of #13882
